src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -277,7 +277,7 @@ object ClusterInitValues {
       clusterImages.find(_.tool == Jupyter).map(_.dockerImage).getOrElse(""),
       clusterImages.find(_.tool == RStudio).map(_.dockerImage).getOrElse(""),
       proxyConfig.jupyterProxyDockerImage,
-      dataprocConfig.welderDockerImage,
+      clusterImages.find(_.tool == Welder).map(_.dockerImage).getOrElse(""),
```

Comment:
I think this was also missed in https://github.com/DataBiosphere/leonardo/pull/979 -- the user-specified welder image needs to be templated to `init-actions.sh`.